### DISCUSSION TITLE
Appreciation thread

### DISCUSSION THREAD

**Discussion by bubbleguuum** [microsoft/WSA#76] [General]
Hi,

I'm a developer that has a non-trivial app in the Amazon appstore since a few years, thus predating WSA by quite a bit.

Since WSA and the Amazon appstore has been available to more users, I've seen a surge in interest for the Amazon version of my app for use under WSA, mainly coming from users already using the Google Play version of my app.

While testing my app under WSA and making minor usability adjustments, I've been impressed by the quality of WSA and how well it works.
I've been delighted that it runs Android 12, with Android 13 planned. 
My app requires other machines on the LAN to be able to connect to the Android VM on sockets, and that's supported via 'Advanced Networking'. 
There are also nice small touches like hoovering the mouse cursor on button displaying tooltips (if they are set in app).

Contrast this with Android on ChromeOS using Android 9 (!!!) or 11 depending on devices, missing inbound network connectivity (which criplples my app) in what seems to be an afterthought, and it is like night and day. Microsoft is doing Android on Windows much better than Google on ChromeOS.

So I'd like to thank the Microsoft WSA team for the incredible work they've done on it (and the new pool of users it opens for Android developers). And it must not have been exactly easy given the beast Android is.

**Comment by nieubank** [comment]
We really appreciate the feedback, thanks for encouraging words!
